### ISSUE TITLE
unlabel dataset

### ISSUE THREAD

**Issue by wm901115nwpu** [yuanyuanli85/Fast_Human_Pose_Estimation_Pytorch#4]
Did unlabel dataset need annotation file?
How arrange the unlabel dataset?

**Comment by yuanyuanli85** [comment]
no. unlabeled dataset does not need any annotation file.   if you want to use training with unlabeled dataset, just create a folder, say data/unlabeled, and put all of images to it.  It is going to be like 1.jpg, 2.jpg under data/unlabeled.  when you train your network, pass it to `--unlabeled_data`. i.e `--unlabeled_data data/unlabeled`

**Comment by wm901115nwpu** [comment]
Ok, I try it.

**Comment by wm901115nwpu** [comment]
When I train hg_s2_b1_mobile_fpd_unlabeled model, if I need mpii dataset?

**Comment by yuanyuanli85** [comment]
yes, you need mpii dataset and unlabeled dataset.  To reproduce that hg_s2_b1_mobile_fpd_unlabeled model, I used 10k images from coco as unlabeled.
